.github/workflows/release.yaml

```diff
@@ -32,6 +32,11 @@ jobs:
           PYPI_PACKAGE_NAME="llm-foundry-test-$(date +%Y%m%d%H%M%S)"
         fi
 
+        # Remove the xentropy-cuda-lib dependency as PyPI does not support direct installs. The
```

Comment:
can you leave a comment in `setup.py` that if someone ever changes that line, they also need to come here and change this line?